src/packageManager.js

```diff
@@ -0,0 +1,35 @@
+"use strict";
```

Comment:
We have `src/plugins/packages/index.js`, which is probably better suited to be there

Reply:
Yeah, good shout. I'll do that when I fix the conflicts for this.